packages/html-reporter/src/testFilesView.tsx

```diff
@@ -67,13 +67,12 @@ export const TestFilesHeader: React.FC<{
   metadataVisible: boolean,
   toggleMetadataVisible: () => void,
 }> = ({ report, filteredStats, metadataVisible, toggleMetadataVisible }) => {
```

Comment:
nit: now that we have a context, we could also replace this props with the context hook